### PR TITLE
Pagination

### LINKED ISSUE #225: Paginate pages
Public:
- published content
- published dbs
- published software

Console:
- Unsubmitted projects
- Submitted projects
- Published projects
- Rejected submissions
- Storage requests
- Users
- News

### PR DESCRIPTION
**Pull request by elfeto**
This closes #225 and paginates the projects, news and past credential applications.

Later if it's wanted, we can add a search for credentialed applications.

### CODE REVIEW

**Comment by tompollard** [comment]
@elfeto do you have any fixtures we can reuse for testing this?

**Comment by elfeto** [comment]
@tompollard I have the same fixtures you gave me.

**Comment by tompollard** [comment]
Okidokes, will give those a try!

File: physionet-django/console/views.py [hunk 1]
```diff
@@ -946,6 +959,7 @@ def news_console(request):
     List of news items
     """
     news_items = News.objects.all().order_by('-publish_datetime')
+    news_items = paginate(request, news_items, 100)
```
Comment (lbulgarelli) [inline physionet-django/console/views.py]: I'd just reduce this pages to 50 items, it was a little bit to much scrolling to get to the bottom (for me).

**Review by lbulgarelli — CHANGES_REQUESTED** [review]

File: physionet-django/console/views.py (hunk 1 above)
Comment (elfeto) [reply physionet-django/console/views.py]: Done

**Review by lbulgarelli — APPROVED** [review]

**Pull request merged**